admin/class-export.php

```diff
@@ -0,0 +1,225 @@
+<?php
+/**
+ * @package    WPSEO
+ * @subpackage Export
+ */
+
+/**
+ * class WPSEO_Export
+ *
+ * Class with functionality to export the WP SEO settings
+ */
+class WPSEO_Export {
+
+	/**
+	 * @var string
+	 */
+	private $export = '';
+
+	/**
+	 * @var string
+	 */
+	public $error = '';
```

Comment:
As far as I can see this property isn't used outside this class, maybe it's better to make it private.